src/dune_analytics.py

```diff
@@ -342,24 +342,20 @@ def open_query(cls, filepath: str) -> str:
 
     def fetch(
             self,
-            query_filepath: str,
+            query_str: str,
             network: Network,
             name: str,
             parameters: Optional[list[QueryParameter]],
     ) -> list[dict]:
         """
-        :param query_filepath: path to sql file to execute
+        :param query_str: path to sql file to execute
```

Comment:
> path to sql file to execute

Is no longer correct if you pass the query itself as a string.

Reply:
gefixed.